htdocs/accountancy/customer/index.php

```diff
@@ -661,6 +671,7 @@
 		}
 		$sql .= " AND f.entity IN (".getEntity('invoice', 0).")"; // We don't share object for accountancy
 		$sql .= " AND f.fk_statut > 0";
+		$sql .= " AND f.fk_statut <> 3"; //Do not calculate the margin of rectified - replaced invoices
```

Comment:
Instead of removing replacement invoice, we should remove the "replaced" invoice ? so  
`AND (close_code IS NULL OR close_code != Facture::CLOSECODE_REPLACED)`

Reply:
Sorry if my English is not very good, I'm Spanish,

 
In "index.php?leftmenu=accountancy_dispatch_customer&mainmenu=accountancy" 
we can see accounting invoices and margins. If there is a replaced invoice, the two are added in margins when only the new invoice that has replaced the erroneous one should be taken into account. In Spain, if there is an error on an invoice, there is up to 4 years to be able to make a corrected invoice.

"When should they be issued?

The rectification will be made as soon as the person obliged to issue the invoice is aware of the circumstances that require its issuance, provided that four years have not elapsed from the moment in which the tax was accrued or, as the case may be, from the date on which that the circumstances regulated in article 80 of Law 37/1992 have occurred, which give rise to the modification of the VAT tax base." 

This is the link in of correctives invoice in spain:
https://sede.agenciatributaria.gob.es/Sede/iva/facturacion-registro/facturacion-iva/facturas-rectificativas.html

Reply:
Its ok.  Works correctly

                $sql .= " AND f.entity IN (".getEntity('invoice', 0).")"; // We don't share object for accountancy
		$sql .= " AND f.fk_statut > 0";
		//$sql .= " AND f.fk_statut <> 3"; //Do not calculate the margin of rectified - replaced invoices
		$sql .= " AND (f.close_code IS NULL OR f.close_code != '".Facture::CLOSECODE_REPLACED."')";
		$sql .= " AND fd.product_type <= 2";

When the pull request is closed, does it mean that it is not taken into account and that it is incorrect? because the calculation of margins works correctly with the previous changes.
Or should I send another with this latest modification? Thanks.